packages/acceptance-tests/pkg-tests-specs/sources/features/__snapshots__/mergeConflictResolution.test.js.snap

```diff
@@ -40,7 +40,7 @@ exports[`Features Merge Conflict Resolution it should properly fix merge conflic
 ➤ YN0000: ┌ Resolution step
 ➤ YN0000: └ Completed
 ➤ YN0000: ┌ Fetch step
-➤ YN0019: │ no-deps-npm-1.0.0-cf533b267a-8bd88a447c.zip appears to be unused - removing
+➤ YN0019: │ no-deps-npm-2.0.0-8c4f3f8395-9c77ddf9a6.zip appears to be unused - removing
```

Comment:
Is this supposed to change / was the old result incorrect?

Reply:
I noticed it but thought the new behaviour made more sense: previously, it seems Yarn was favouring the remote changes over the local ones; thinking about it, the opposite feels more sensible to me, because otherwise we're may be overwriting the changes that the local user was working on (for example if they upgraded a package).

That being said, I'm not sure any of these options is perfect anyway; there are probably situation where they'd each be "the best".